corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -108,7 +108,17 @@
                             </th>
                             <!-- /ko -->
                             {% if commtrack_enabled %}
-                            <th>{% trans "Tracks Stock" %}</th>
+                            <th>
+                                {% trans "Tracks Stock" %}
+                                <span class="hq-help-template"
+                                      data-title="{% trans "Tracks Stock" %}"
+                                      data-content="{% blocktrans %}
+                                          Does this location keep inventory and track stock?
+                                          Note that changing this property will update all
+                                          locations of this level accordingly.
```

Comment:
might want to get @sheelio to review this. the second sentence seems kind of vague/confusing to me